libfwupd/fwupd-client.c

```diff
@@ -84,6 +88,18 @@ static guint signals [SIGNAL_LAST] = { 0 };
 G_DEFINE_TYPE_WITH_PRIVATE (FwupdClient, fwupd_client, G_TYPE_OBJECT)
 #define GET_PRIVATE(o) (fwupd_client_get_instance_private (o))
 
+static void
+fwupd_client_curl_helper_free (FwupdCurlHelper *helper)
```

Comment:
Should this early-return if `helper == NULL`, mimicking `g_free` behavior?

Reply:
I believe G_DEFINE_AUTOPTR_CLEANUP_FUNC does that for us.

Reply:
Absolutely true, I should have double-checked the docs first, sorry for the noise.

Reply:
Not noise! I really appreciate the sanity checks!